GOVERNANCE.md

```diff
@@ -160,6 +160,30 @@ The roles used in this document are custom roles mapped according to the [GitHub
 
 If any of the above roles hasn't contributed in any phases (including, but not limited to: code changes, doc updates, issue discussions) in 3 months, the administrator needs to inform the member and remove one's roles and GitHub permissions.
 
+## Maintainers Areas
+
+The Kyverno code base covers many areas and project maintainers are not required to know everything about the project.
+For this reason, maintainers are specific to one (or more) area of the code base.
+Every area represent a specific aspect of the project.
+
+### Kyverno areas
```

Comment:
I also recommend list the sub-projects for completeness:
- Kyverno JSON
- Kyverno Chainsaw
- Kyverno Policy Reporter

We don't have to have detailed categories in these.

Reply:
@JimBugwadia shall i add playground and reports-server ?